platform/barefoot/libsaithrift-dev.mk

```diff
@@ -2,19 +2,19 @@
 
 SAI_VER = 0.9.4
 
-LIBSAITHRIFT_DEV = libsaithrift-dev_$(SAI_VER)_amd64.deb
+LIBSAITHRIFT_DEV = libsaithrift-dev_$(SAI_VER)_$(CONFIGURED_ARCH).deb
 $(LIBSAITHRIFT_DEV)_SRC_PATH = $(SRC_PATH)/sonic-sairedis/SAI
-$(LIBSAITHRIFT_DEV)_DEPENDS += $(LIBTHRIFT) $(LIBTHRIFT_DEV) $(PYTHON_THRIFT) $(THRIFT_COMPILER) $(BFN_SAI) $(BFN_SAI_DEV)
-$(LIBSAITHRIFT_DEV)_RDEPENDS += $(LIBTHRIFT) $(BFN_SAI)
+$(LIBSAITHRIFT_DEV)_DEPENDS += $(LIBTHRIFT_0_13_0) $(LIBTHRIFT_DEV_0_13_0) $(PYTHON_THRIFT_0_13_0) $(THRIFT_COMPILER_0_13_0) $(LIBSAIMETADATA_DEV) #$(LIBSAIVS) $(LIBSAIVS_DEV) $(LIBSAIMETADATA)
```

Comment:
Is there a dependency on thrift 0.13 for the SAI PTF?

Reply:
Actually, thrift 0.13.0 is backward compatible with 0.11.0 in almost all cases, but the original thrift target is built with an additional option [`-DFORCE_BOOST_SMART_PTR` ](https://github.com/Azure/sonic-buildimage/blob/master/src/thrift/Makefile#L30). This causes issues for BFN SDE as we use thrift in driver, platform, and also, in SAIv2 rpc server and everywhere with  `std::smart_ptr`. To fix those issues I have added thrift_0_13_0 target to sonic-buildimage recently https://github.com/Azure/sonic-buildimage/pull/8307. That is why `LIBSAITHRIFT_DEV` depends on `LIBTHRIFT_0_13_0`


UPD: `-DFORCE_BOOST_SMART_PTR` was removed in thrift 0.13.0 and `std::smart_ptr` is used by default.